src/main/java/org/opensearch/security/resources/api/share/ShareRequest.java

```diff
@@ -186,6 +201,7 @@ public void parseContent(XContentParser xContentParser, ResourcePluginInfo resou
                                 this.resourceId(parser.text());
                                 break;
                             case "resource_type":
+                                this.resourceType(parser.text());
```

Comment:
Nice, I started making similar change on https://github.com/cwperks/security/pull/66 to change the restriction of resource type <-> resource index being 1-to-1 to become many <-> 1